common.css

```diff
@@ -2,6 +2,7 @@
 @import url('/Pages/ProductPage/product.css');
 @import url('/Pages/LoginPage/login.css');
 @import url('/Pages/SignupPage/signup.css');
+@import url('/Pages/CartPage/cart.css');
 
 .main-wrapper{
     display: flex;
```

Comment:
that class can be used directly in this element.

Reply:
ok thanks for the suggestions i will look into them